data/flexibleForms/adultCaseNote.ts

```diff
@@ -0,0 +1,189 @@
+import { Form } from './forms.types';
+import { format } from 'date-fns';
+
+const form: Form = {
+  id: 'adult-case-note',
+  name: 'Case note',
+  groupRecordable: true,
+  isViewableByAdults: true,
+  isViewableByChildrens: false,
+  canonicalUrl: (socialCareId) => `/people/${socialCareId}/case-note`,
+  steps: [
+    {
+      id: 'Case note',
+      name: 'Case note',
+      theme: 'Case note',
+      fields: [
+        {
+          id: 'Type',
+          question: 'What kind of note is this?',
+          type: 'radios',
+          required: true,
+          choices: [
+            {
+              value: 'Visit',
+              label: 'Visit',
+            },
+            {
+              value: 'Correspondence',
+              label: 'Correspondence',
+            },
+            {
+              value: 'Something else',
+              label: 'Something else',
+            },
+          ],
+        },
+        {
+          id: 'Correspondence type',
+          question: 'What kind of correspondence?',
+          type: 'radios',
+          required: true,
+          condition: {
+            id: 'Type',
+            value: 'Correspondence',
+          },
+          choices: [
+            {
+              value: 'Phone call',
+              label: 'Phone call',
+            },
+            {
+              value: 'Email, letter or text message',
+              label: 'Email, letter or text message',
+            },
+          ],
+        },
+        {
+          id: 'Visit type',
+          question: 'What kind of visit?',
+          type: 'radios',
+          required: true,
+          condition: {
+            id: 'Type',
+            value: 'Visit',
+          },
+          choices: [
+            {
+              value: 'Home',
+              label: 'Home visit',
+            },
+            {
+              value: 'Office',
+              label: 'Office visit',
+            },
+            {
+              value: 'Virtual',
+              label: 'Virtual visit',
+            },
+            {
+              value: 'No reply to home visit',
+              label: 'No reply to home visit',
+            },
+          ],
+        },
+
+        {
+          id: 'Topics',
+          question: 'Topics',
+          hint: 'Help colleagues find this note. Add as many as you need.',
+          type: 'checkboxes',
+          condition: {
+            id: 'Type',
+            value: 'Something else',
+          },
+          choices: [
```

Comment:
taken from current live list, with duplicates from radios above removed.

Reply:
I don't think it will be possible to query by these with the current setup. I do have some thoughts around tagging/topics and how we can do it though.